src/Nelmio/Alice/Exception/Instantiator/ExceptionInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *  
+ *  (c) Nelmio <hello@nelm.io>
+ *  
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace Nelmio\Alice\Exception\Instantiator;
+
+/**
+ * @author Théo FIDRY <theo.fidry@gmail.com>
+ */
+interface ExceptionInterface extends \Throwable
```

Comment:
The name might be confusing:

- When using it without aliases, you have `ExceptionInterface`, `InstantiatorExceptionInterface` would maybe be clearer
- It's weird to have `Exception` when technically speaking it's a `Throwable`, should be renamed to reflect on that?